docs/integrations/pytorch_lightning.rst

```diff
@@ -1,318 +1,315 @@
-Neptune-PyTorch Lightning Integration
-=====================================
+PyTorch Lightning Integration
+=============================
 
-The integration enables you to log |PyTorch Lightning| metrics to Neptune.
-
-.. image:: ../_static/images/integrations/pytorchlightning_neptuneml.png
-   :target: ../_static/images/integrations/pytorchlightning_neptuneml.png
+What will you get?
+------------------
+.. image:: ../_static/images/integrations/lightning_basic.gif
+   :target: ../_static/images/integrations/lightning_basic.gif
    :alt: PyTorchLightning neptune.ai integration
 
-Requirements
-------------
-This integration is enabled as part of PyTorch Ligthning loggers module, so all you need is to have |neptune-client| installed.
+PyTorch Lightning is a lightweight PyTorch wrapper for high-performance AI research. With Neptune integration you can:
+
+* see experiment as it is running,
+* log training and testing metrics, and visualize them in Neptune UI,
+* log experiment parameters,
+* monitor hardware usage,
+* log any additional metrics of your choice,
+* log performance charts,
+* save model checkpoints.
+
+.. note::
+
+    This integration is tested with ``pytorch-lightning==0.9.0``. See 'Troubleshoot' section for information about other versions.
+
+Quickstart
+----------
+This quickstart will show you how to log PyTorch Lightning experiments to Neptune using ``NeptuneLogger`` (part of the pytorch-lightning library).
+
+Check this public project with example experiments: |project|.
```

Comment:
I must have miscoummunicated but I think:
- project
- script-basic (maybe script-advanced)
- colab-basic (maybe colab-advanced)

Should be a part of the what will you get or we can have a section with some other name above quickstart and below what will you get. 

Definitely |project| shouldn't be the first thing here. It doesn't help the person go through quickstart does it?
It can show what you get nicely -> hence I feel it makes sense to have it above in what you will get. 

When it comes to script and colab the reason for putting it up there (not sure if my "implementation" is good) is to show people right of the bat that there is a code that will work that they can use (come back to)

Reply:
ok, cool, corrected.